src/neo/Network/P2P/Payloads/Header.cs

```diff
@@ -204,6 +217,7 @@ public JObject ToJson(ProtocolSettings settings)
             json["previousblockhash"] = prevHash.ToString();
             json["merkleroot"] = merkleRoot.ToString();
             json["time"] = timestamp;
+            json["nonce"] = nonce.ToString("X");
```

Comment:
Could you tell me why you convert it to a hex string, not directly a decimal string?

Reply:
lol, cause i think a decimal string looks weird....and is longer than a hex string.

Reply:
```suggestion
            json["nonce"] = nonce.ToString("X8");
```

Maybe a fixed 8-digit number would be better.